### ISSUE TITLE
Update CLI `--help` text

### ISSUE THREAD

**Issue by w0whitaker** [11ty/eleventy#3230]
Run `npx @11ty/eleventy --help` from the command line and the resulting list includes the following:
```
--dryrun
       Don’t write any files. Useful with `DEBUG=Eleventy* npx eleventy`
```

It's a very minor thing, but I believe it should be `DEBUG=Eleventy* npx @11ty/eleventy` to be consistent with the documentation.

I'd be more than happy to work on this soon as I figure out where the text is produced.

**Comment by baptiste-roullin** [comment]
The text used in the help [is here](https://github.com/11ty/eleventy/blob/8f95fac1c4220d2b19dbc959861cd3d3e3650a21/src/Eleventy.js#L679).

**Comment by w0whitaker** [comment]
> The text used in the help [is here](https://github.com/11ty/eleventy/blob/8f95fac1c4220d2b19dbc959861cd3d3e3650a21/src/Eleventy.js#L679).

Thank you, I've submitted a PR with new wording.

**Comment by zachleat** [comment]
Awesome, thank you!

If someone else wanted to contribute changing that command when we know we’re on Windows that would be a further improvement!

Example on https://www.11ty.dev/docs/debugging/

**Comment by zachleat** [comment]
Shipped with 3.0.0-alpha.6
